execution/gethexec/express_lane_service.go

```diff
@@ -31,9 +31,14 @@ type expressLaneControl struct {
 	controller common.Address
 }
 
+type transactionPublisher interface {
+	publishTransactionImpl(context.Context, *types.Transaction, *arbitrum_types.ConditionalOptions, bool) error
```

Comment:
* an interface with private function can get weird if we want to move code around.
* an interface should make sense as itself and func-names should self-document
* I think you only ever pass "true" in that bool, is that right?
Mybe have something like "PublishFastLaneTransaction" or similar that doesn't accept the bool and is trivially implemented in the sequencer or a wrapper?